docs/kpod-stop1.md

```diff
@@ -0,0 +1,35 @@
+% kpod(1) kpod-stop - Stop one or more containers
+% Brent Baude
+# kpod-stop "1" "September 2018" "kpod"
+
+## NAME
+kpod stop - Stop one or more containers
+
+## SYNOPSIS
+**kpod** **stop** [*options* [...]] container
+
+## DESCRIPTION
+Kpod stop will stop one or more containers.  You may use container IDs or names as input.  The **--timeout** switch
+allows you to specific the number of seconds to wait before forcibly stopping the container after the stop command 
+is issued to the container.  The default is 10 seconds.
+
+## OPTIONS
+
+**--timeout, t**
+
+Timeout to wait before forcibly stopping the container
+
+
+## EXAMPLE
+
+kpod stop mywebserver
+
+kpod stop 860a4b23
+
+kpod stop --timeout 2 860a4b23
+
+## SEE ALSO
+kpod(1), kpod-rm(1)
+
+## HISTORY
+September 2018, Originally compiled by Brent Baude <bbaude@redhat.com>
```

Comment:
Me thinks you're a little ahead of yourself or are anticipating a very long PR process!  2017 maybe?  ;^)

Reply:
fixed